apps/roam/src/index.ts

```diff
@@ -126,7 +127,18 @@ export default runExtension(async (onloadArgs) => {
   document.addEventListener("input", discourseNodeSearchTriggerListener);
   document.addEventListener("selectionchange", nodeCreationPopoverListener);
 
-  await initializeSupabaseSync();
+  const supabase = createClient();
+  if (supabase) {
+    const { data } = await supabase
```

Comment:
This will fail unless you use a logged in client.

Reply:
Realizing this is not just a sanity check, but probably used as a proxy for whether the embeddings were activated. But the Space entry is not accessible before login, and is created as a side-effect of login. So that information should be stored roam-side. (Having a sanity check for database access still makes sense...)